doc/sphinx/installation/install_source.rst

```diff
@@ -166,9 +166,11 @@ dependencies:
 	docutils \
 	sphinx-doc
 
-* Add sphinx to PATH as advised by the installer::
+* Add the sphinx tools and the docutils to PATH so `configure` will find them.
 
-    PATH="/usr/local/opt/sphinx-doc/bin:$PATH"
+    PATH="/opt/homebrew/opt/sphinx-doc/bin:/opt/homebrew/opt/docutils/bin:$PATH"
+
+It'll be a good idea to persist these changes .0m.so you can rebuild the source later.
```

Comment:
What is this ".0m." doing in the middle of the sentence?

Reply:
No idea. I have some problems with my terminal acting up.